web3_proxy/src/frontend/users/rpc_keys.rs

```diff
@@ -41,11 +42,33 @@ pub async fn rpc_keys_get(
         .await
         .web3_context("failed loading user's key")?;
 
+    let secondary_user_entities = secondary_user::Entity::find()
+        .filter(secondary_user::Column::UserId.eq(user.id))
+        .all(db_replica.as_ref())
+        .await?
+        .into_iter()
+        .map(|x| (x.rpc_secret_key_id, x))
+        .collect::<HashMap<u64, secondary_user::Model>>();
+
+    // Now return a list of all subusers (their wallets)
+    let rpc_key_entities: Vec<rpc_key::Model> = rpc_key::Entity::find()
+        .filter(
+            rpc_key::Column::Id.is_in(
+                secondary_user_entities
+                    .iter()
+                    .map(|(x, _)| *x)
```

Comment:
nit: Instead of `iter().map(...)`, I think this can be `.keys().copied()`

I'm also surprised it needs to `collect` and can't just take an iter